File: JKWKWebViewHandler_OC/Example/JKWKWebViewHandler/JKPluginA.m
//
//  JKPluginA.m
//  JKWKWebViewHandler_Example
//
//  Created by JackLee on 2019/7/21.
//  Copyright © 2019 HHL110120. All rights reserved.
//

#import "JKPluginA.h"

@implementation JKPluginA
+ (void)getNativeInfo:(NSDictionary *)params :(void(^)(id response))successCallBack :(void(^)(id response))failureCallBack{
    NSLog(@"getNativeInfo %@",params);
    if (successCallBack) {
        successCallBack(@"success !!!");
    }
    if (failureCallBack) {
        failureCallBack(@"failure !!!");
    }
}
@end

File: JKWKWebViewHandler_Swift/Example/JKWKWebViewHandler_Swift/JKPluginA.swift
//
//  JKPluginA.swift
//  JKWKWebViewHandler_Swift_Example
//
//  Created by JackLee on 2021/1/26.
//  Copyright © 2021 CocoaPods. All rights reserved.
//

import Foundation
class JKPluginA: NSObject {
 class func getNativeInfo(params:Dictionary<String,Any>, successCallBack:((_ response:Any?) -> Void)?, failureCallBack:((_ response:Any?) -> Void)?) -> Void {
        if successCallBack != nil {
            successCallBack!("success !")
        }
        
        if failureCallBack != nil {
            failureCallBack!("failure !")
        }
    }
}
